stable/chartmuseum/requirements.yaml

```diff
@@ -0,0 +1,4 @@
+dependencies:
+- name: common
+  version: 0.0.1
+  repository: https://kubernetes-charts-incubator.storage.googleapis.com/
```

Comment:
https://github.com/mattfarina/charts/blob/c2244a78730c600254428d64d9b1195907103d81/PROCESSES.md#promoting-a-chart-from-incubator-to-stable states that stable charts cannot depend on incubator charts. Not sure if we're ready to move the common chart to stable /cc @prydonius, else you'd need to remove this dependency @jdolitsky .

Reply:
Indeed stable charts should not depend on incubator charts. The common chart is not in stable yet, the main reason being that it isn't a normal chart in its own right and will not pass CI. We've previously discussed creating a new chart repo (e.g. tooling or libraries) to host the common chart.

We have a few options here:
1. Keep ChartMuseum in incubator whilst it relies on the incubated common.
1. Make an exception for the common chart, allowing all stable charts to include it.

We have to separately figure out what we want to do with the common chart, I think we need to move it away from incubator.

Reply:
We chatted about this during the Charts Office Hours and decided that until we make more of an effort to bring the common chart to stable (specifically we need to work on updating API versions, come up with a way to test it), this chart either needs to stay in incubator or remove the common chart dependency.

Reply:
@prydonius ok thats fine. Will keep in incubator for now, no plan to remove common dependency. Closing.